templates/deb/postinst_upgrade.sh.erb

```diff
@@ -1,15 +1,21 @@
 #!/bin/sh
 after_upgrade() {
+<%# Making sure that at least one command is in the function -%>
+<%# avoids a lot of potential errors, including the case that -%>
+<%# the script is non-empty, but just whitespace and/or comments -%>
```

Comment:
My agreement, for posterity: We could also just sanitize ensure at least one character is non whitespace, but I think it's just safer to prepend `:` as the first command.